lib/execution_engine2/models/models.py

```diff
@@ -126,7 +126,8 @@ class Job(Document):
     scheduler_type = StringField()
     scheduler_id = StringField()
     job_input = EmbeddedDocumentField(JobInput, required=True)
-    job_output = EmbeddedDocumentField(JobOutput)
```

Comment:
I have zero clue what's going on here. Just like `LogLines` I will get validation error if I keep it as `EmbeddedDocumentField`

`mongoengine.errors.ValidationError: ValidationError (Job:5d538097c2a21e95b5f8dc7c) (Invalid embedded document instance provided to an EmbeddedDocumentField: ['job_output'])`

Reply:
Yea it's weird. JobInput is working right?

Reply:
yup so strange. JobInput works.